test/e2e/gameserverallocation_test.go

```diff
@@ -85,9 +85,9 @@ func TestMultiClusterAllocationOnLocalCluster(t *testing.T) {
 					Priority: 1,
 					Weight:   100,
 					ConnectionInfo: multiclusterv1alpha1.ClusterConnectionInfo{
-						AllocationEndpoint: "localhost",
-						ClusterName:        "multicluster1",
-						SecretName:         "sec1",
+						AllocationEndpoints: []string{"localhost"},
```

Comment:
Should we have test with multiple AllocationEndpoints as well?

Reply:
Thanks. I have added component testing for this case. Adding end-to-end testing requires fault injection. I'll add it in future, when the testing infrastructure is more mature to allow for that. For now we should be fine.